java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JwtX5tValidator.java

```diff
@@ -29,11 +30,10 @@
 public class JwtX5tValidator implements Validator<Token> {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(JwtX5tValidator.class);
-	private final OAuth2ServiceConfiguration config;
-
+	static final String VALIDATION_FAILED = "Certificate cnf token validation failed";
 
 	public JwtX5tValidator(OAuth2ServiceConfiguration config) {
-		this.config = config;
+		Assertions.assertNotNull(config, "config must not be null");
```

Comment:
I would prefer a bit more detailed message like "OAuth2ServiceConfiguration must not be null"

Reply:
updated to "No token passed to validate certificate thumbprint"

Reply:
rephrased it to "Service configuration must not be null" as on other places.